backend/src/main/java/kr/momo/service/meeting/MeetingConfirmService.java

```diff
@@ -94,4 +99,33 @@ private void validateDateRange(Meeting meeting, LocalDateTime startDateTime, Loc
             throw new MomoException(MeetingErrorCode.INVALID_DATETIME_RANGE);
         }
     }
+
+    @Transactional
```

Comment:
현재 해당 로직은 조회만 하는 로직인 것 같습니다. 옵션을 readOnly로 바꿔보는 것은 어떨까요?
```suggestion
    @Transactional(readOnly = true)
```